andela/vidar-ah-frontend
# #165034251 Setup CircleCI for continuous integration

#### What does this PR do?
This PR integrates CircleCI with the repository
#### Desciption of task to be complete?
* Add circleci config file
* Add badge to readme
#### How can this be manually tested?
* Push code to this repo, a build is automatically generated
* Check the readme file for the build status
#### Any background context you would like to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165034251
